### PR TITLE
RIFEX-209/210/213 Loading states for channel actions and SDK actions

### PR DESCRIPTION
**Pull request by debrayperalta**
This PR aims to improve the UX of the Open, Deposit and Close operations on the Lumino channels

- Now we display an appropiate message explaining what we are doing
- The message includes an explanation that the operation may take X minutes
- If the SDK has support for Temporary Channels (when a channel has not been opened) it will be displayed alongside others
- The Close actions now have a spinner and a proper message

### CODE REVIEW

**Review by rorolopetegui — APPROVED** [review]
lgtm

**Review by marcosmartinez7 — APPROVED** [review]

File: ui/app/rif/components/lumino/open-channel/index.js [hunk 1]
```diff
@@ -206,7 +208,7 @@ class OpenChannel extends Component {
       if (this.state.amount) {
         this.setState({
           loading: true,
-          loadingMessage: 'Making deposit...',
+          loadingMessage: 'Making deposit\nThis operation can take around 6 minutes',
```
Comment (JONAF2103) [inline ui/app/rif/components/lumino/open-channel/index.js]: I think a better way to say that could be 'Making deposit.\n Please wait, this operation could take about 6 minutes.', but it's just a suggestion.

File: ui/app/rif/components/lumino/open-channel/index.js [hunk 2]
```diff
@@ -268,7 +269,7 @@ class OpenChannel extends Component {
           }
           this.setState({
             loading: true,
-            loadingMessage: 'Opening...',
+            loadingMessage: 'Opening channel\nThis operation can take around 4 minutes',
```
Comment (JONAF2103) [inline ui/app/rif/components/lumino/open-channel/index.js]: Same thing here, it's better if we use please and about, more user friendly :)

File: ui/app/rif/pages/rns/lumino-channels/index.js [hunk 3]
```diff
@@ -179,49 +184,66 @@ class LuminoChannels extends Component {
     }
     return retVal;
   }
+
+  closingMessage = () => {
+    // eslint-disable-next-line camelcase
+    const {channel: {sdk_status}} = this.props;
+    const messages = {
+      'CHANNEL_WAITING_FOR_CLOSE': 'Closing channel\nThis operation can take around 4 minutes',
```
Comment (JONAF2103) [inline ui/app/rif/pages/rns/lumino-channels/index.js]: Same here as above

**Review by JONAF2103 — APPROVED** [review]
LGTM i suggest to just make some minor adjustments to the texts, like less robot and more user friendly.

**Pull request merged**